packages/babel-plugin-proposal-do-expressions/test/fixtures/do-expression-switch-case/condition-before-break/input.js

```diff
@@ -0,0 +1,10 @@
+const x = (n) => do {
+  switch (n) {
+    case 0:
+      if (true) {
+        'out';
+        break;
+      }
```

Comment:
Can you add a test case where `BreakStatement` does not follow an `ExpressionStatement`?
```js
case 1:
  if (true) {
    break;
  }

```